.ci/build_core.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+# Build and install the PSL core matching this version.
+BUILD_DIR='/tmp/__building_psl_core__'
+TARGET_REPOS="$@"
+
+if [ $# -eq 0 ]; then
+   echo "No target repositories specified"
+   exit 1
+fi
+
+# If the pushed branch is master, we will use master for core.
+# Otherwise, we will use develop.
+branch='develop'
+if [ "$TRAVIS_BRANCH" = 'master' ]; then
+   branch='master'
+fi
+
+# We will always use the same owner for PSL core as this repo's owner.
+owner=$(echo "$TRAVIS_REPO_SLUG" | sed 's#/.\+$##')
+
+# Make an exception for linqs user's develop branch as most development
```

Comment:
Be more direct in this comment.
Just something like: "linqs development is done under the eriq-augustine fork."